packages/slate/src/models/operation.js

```diff
@@ -124,10 +125,6 @@ class Operation extends Record(DEFAULTS) {
         v = Range.create(v)
       }
 
-      if (key == 'value') {
-        v = Value.create(v)
-      }
-
```

Comment:
Is removing this part required? It looks like the other skipped properties are still present in this section.

Reply:
Ah, I read that as dead code. But I guess it would still run if you specifically passed something for the `value` key.